### DISCUSSION TITLE
Invoice Notifikasi lewat WhatsApp

### DISCUSSION THREAD

**Discussion by Bisbissdev** [kirimwa-id/forum#65] [Pamerkan]
Saya sedang develop sebuah website untuk penjualan buku dan untuk invoice saya gunakan API dari kirimWa, sampai sejauh ini hasilnya sesuai dengan kemauan. Terimakasih mas Rio ditunggu update fitur terbarunya..
![0758BA57-2B32-42D4-8FCD-B15AF043EF33](https://user-images.githubusercontent.com/93394155/140587303-8e798493-0a96-4762-bf8b-19a8e851b97f.jpeg)

**Comment by rioastamal** [comment]
Terima kasih @Bisbissdev atas testimoni penggunaan API KirimWA.id 👍 , terus ikuti forum ini untuk update selanjutnya.
